telemetry/telemetry.proto

```diff
@@ -4,6 +4,57 @@ import 'nanopb.proto';
 
 option java_outer_classname="TelemetryProto";
 
-message Telemetry {
-  optional int32 timestamp_ms = 1;
+//Should be sent only after device resets
```

Comment:
This is just firmware (assuming the receive pin from the air interface is connected), some way for the telemetry viewer to query the telemetry board for metadata.

Or, as discussed on slack, metadata could be encoded elsewhere and available offline to both the firmware and telemetry viewer.